src/modules/layout/components/Header.js

```diff
@@ -120,22 +121,29 @@ class Header extends React.PureComponent<PropsType, StateType> {
   }
 
   goToLanguageChange = () => {
-    this.getNavigation().navigate('ChangeLanguageModal')
+    this.getNavigation().navigate({
+      routeName: 'ChangeLanguageModal',
+      params: {
+        availableLanguages: this.props.availableLanguages
+      }
+    })
   }
 
   render () {
+    const { theme, availableLanguages } = this.props
     if (this.state.searchActive) {
-      return <BoxShadow theme={this.props.theme}><HorizontalLeft>
+      return <BoxShadow theme={theme}><HorizontalLeft>
         <HeaderBackButton onPress={this.closeSearchBar} />
-        <ThemedSearchBar theme={this.props.theme} />
+        <ThemedSearchBar theme={theme} />
       </HorizontalLeft>
       </BoxShadow>
     }
 
     const headerTitle = this.getDescriptor().headerTitle || ''
+    const isOtherLanguageAvailable = availableLanguages && availableLanguages.length > 0
```

Comment:
why do you need to check for `availableLanguages` to be truthy? The proptype suggests it to be always truthy.

Reply:
The prop types are wrong, it isn't always truthy.